client/src/constants.ts

```diff
@@ -6,3 +6,7 @@ export const CRUD_MODE = JSON.parse(
   process.env.REACT_APP_CRUD_MODE ||
     JSON.stringify(process.env.NODE_ENV === "development")
 );
+
+export const NO_WATCHER = JSON.parse(
+  process.env.REACT_APP_NO_WATCHER || "false"
```

Comment:
This also needs to be mentioned in [`docs/envvars.md`](https://github.com/mdn/yari/blob/master/docs/envvars.md#client).

Reply:
Bah! Ignore me. I read this PR wrong.